cas-server-support-saml-idp/build.gradle

```diff
@@ -0,0 +1,143 @@
+description = 'Apereo CAS SAML Identity Provider Support'
+dependencies {
+    compile project(':cas-server-core-services')
+    compile(project(':cas-server-support-saml')) {
+        exclude(module: 'xml-apis')
+        exclude(module: 'guava')
+    }
+
+    compile group: 'org.apache.commons', name: 'commons-lang3', version:commonsLangVersion
+    compile(group: 'org.jasig.cas.client', name: 'cas-client-core', version: casClientVersion) {
+        exclude(module: 'servlet-api')
+        exclude(module: 'log4j-over-slf4j')
+        exclude(module: 'bcprov-jdk15')
+        exclude(module: 'slf4j-api')
+    }
+    compile(group: 'net.shibboleth.idp', name: 'idp-core', version: idpVersion) {
```

Comment:
Yes. This does make it easy on them. What is happening with the build is that gradle by default is configured to do very strict dependency matching and filtering, which means transitive deps that conflict with ours are automatically stopped and build failed. Just so we can catch weird issues later on, but it does make the build uglier. I prefer that than having a deployer make sense out of `ClassNotFoundException` errors.